sceptre/connection_manager.py

```diff
@@ -214,15 +218,15 @@ def _get_client(self, service, region, profile, stack_name, iam_role):
                     "No %s client found, creating one...", service
                 )
                 self._clients[key] = self._get_session(
-                    profile, region, iam_role
+                    profile, region, iam_role, duration_seconds
```

Comment:
I think you'd want to use `self.duration_seconds` for this rather than needing it passed as a parameter.